run_dir/static/js/running_notes.js

```diff
@@ -147,20 +148,66 @@ function filter_running_notes(search){
         }
     });
 }
+//Count rn's of each category and add badge with number to filter dropdown
+function count_cards(){
+    var cat_cards = {};
+    var all = 0;
+    $('#running_notes_panels').find('.badge').each(function(){
+        var label = $.trim($(this).text());
+        all++
+        if (label){
+            if (label in cat_cards){
+                cat_cards[label]++;
+            }else{
+                cat_cards[label] = 1;
+            }
+        }
+    });
+    $('.btn_count').append('All'+'&nbsp;'+'<span class="badge bg-secondary">'+all+'</span>');
+    $('#rn_category').next().find('.dropdown-item').each(function(){
+        var label = $.trim($(this).text())
+        if (label == 'All'){
+            $('.all_count').prepend('<span class="badge bg-secondary">'+all+'</span>'+'&nbsp;');
+        }else if (!cat_cards[label]){
+            $(this).parent('li').addClass('d-none');
+        }else{
+            if (label == 'Workset'){
+                $(this).prepend('<span class="badge bg-primary">'+cat_cards[label]+'</span>'+'&nbsp;');
+            }else if (label == 'Flowcell'){
+                $(this).prepend('<span class="badge bg-success">'+cat_cards[label]+'</span>'+'&nbsp;');
+            }else if (label == 'Decision'){
+                $(this).prepend('<span class="badge bg-info">'+cat_cards[label]+'</span>'+'&nbsp;');
+            }else if (label == 'Lab'){
+                $(this).prepend('<span class="badge bg-succe">'+cat_cards[label]+'</span>'+'&nbsp;');
+            }else if (label == 'Bioinformatics'){
+                $(this).prepend('<span class="badge bg-warning">'+cat_cards[label]+'</span>'+'&nbsp;');
+            }else if (label == 'User Communication'){
+                $(this).prepend('<span class="badge bg-usr">'+cat_cards[label]+'</span>'+'&nbsp;');
+            }else if (label == 'Administration'){
+                $(this).prepend('<span class="badge bg-danger">'+cat_cards[label]+'</span>'+'&nbsp;');
+            }else if (label == 'Important'){
+                $(this).prepend('<span class="badge bg-imp">'+cat_cards[label]+'</span>'+'&nbsp;');
+            }else if (label == 'Deviation'){
+                $(this).p('<span class="badge bg-devi">'+cat_cards[label]+'</span>'+'&nbsp;')
+            }
+        }
+    });
+}
+
 //Filter notes
 $('#rn_search').keyup(function() {
     var search=$('#rn_search').val();
     filter_running_notes(search);
 });
 
-$(document).ready(function() {
-    $('#rn_category').change(function() {
-        var search=$('#rn_category :selected').text();
-        if (search == 'All'){
-            search='';
-        }
-        filter_running_notes(search);
-    });
+//Filter dropdown
+$('.note_category').on('click', function(){
+    $('#rn_category').html($(this).html());
+    var search = $(this)[0].textContent.replace(/[0-9]/g, '');
```

Comment:
This could be 
 `var search = this.lastChild.textContent;`

Reply:
Nice!